keyboards/claw44/lib/layer_state_reader.c

```diff
@@ -0,0 +1,35 @@
+
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+#include "claw44.h"
+
+#define L_BASE 0
+#define L_LOWER 2
```

Comment:
To make this more universal,  you can change this: 
```suggestion
#define L_LOWER (1<<_LOWER)
```